package.json

```diff
@@ -8,40 +8,40 @@
     "email": "justin@testdouble.com",
     "url": "http://testdouble.com"
   },
-  "main": "index.js",
-  "browser": "lib/testdouble.coffee",
```

Comment:
nope. the browser entry was only used to bypass the coffeescript registration being done in index.js. now that there is no coffeescript registration, the node entrypoint and browser entrypoint are the same.